autoPyTorch/pipeline/components/training/trainer/base_trainer.py

```diff
@@ -20,6 +21,22 @@
 from autoPyTorch.utils.implementations import get_loss_weight_strategy
 
 
+class StepIntervalUnit(Enum):
```

Comment:
hey as this class is something that is used in lr schedulers as well and more importantly, this is something to do with the lr schedulers. Do you think it makes sense to move it to a utils file of the lr_scheduler folder?

Reply:
It makes sense
I will do it